entry_types/scrolled/spec/controllers/pageflow_scrolled/entries_controller_spec.rb

```diff
@@ -11,6 +11,7 @@ module PageflowScrolled
     describe '#show' do
       it 'renders script tag for frontend js' do
         entry = create(:entry, :published)
+        create(:scrolled_chapter, revision: entry.published_revision)
```

Comment:
This looks like it might either be not needed or hiding a code path that cannot handle entries without chapters.

Reply:
This was added to fix the linting spec for new entries. I removed it in favor of building a placeholder storyline in https://github.com/codevise/pageflow/pull/1288/commits/3dbbf34a19651a4b21c8be2ee507b8d492410620#diff-4f20f4b7f92dc9e9012f90518a9b90edR20